lib/active_admin/resource.rb

```diff
@@ -135,6 +135,10 @@ def belongs_to_config
       @belongs_to ||= []
     end
 
+    def belongs_to_params
+      belongs_to_config.select{ |c| c.required? }.map{|c| c.to_param }
```

Comment:
@timoschilling Isn't @houndci-bot supposed to make this kind of comments? Or is it disabled?